src/load.py

```diff
@@ -52,14 +41,12 @@ def lambda_handler(event, context):
         logger.debug(event)
         size = int(event['Record']['s3']['object']['size'])
         if size < int(os.getenv('S3_OBJECT_SIZE_LIMIT', 150000000)):
-            logger.debug(f"Processing small S3 object {size}")
             record_id = etl(event)
         else:
-            logger.debug(f"Processing large S3 object {size}")
-            record_id = etl_big(event)
+            raise Exception(f"File too large to process for event {event}")
```

Comment:
If we're going to have the initial load throw this exception, I assume then it's no longer the trigger's responsibility to keep the large files from coming in. I'd like to suggest as a follow up to this, that https://github.com/usgs/aqts-capture-trigger/blob/master/serverless.yml#L49 be removed.